### ISSUE TITLE
no link to the adjusted ISTD dataset?

### ISSUE THREAD

**Issue by nachifur** [cvlab-stonybrook/SID#7]
Hello, your work is very exciting, thank you for your contribution to this field.
But I can’t find where to download your adjusted ISTD dataset? Why is there no link to the adjusted data set in the readme file, but only the test results? Can you provide the adjusted training set and test set? 
Thank you for your work and look forward to your reply.

**Comment by hieulem** [comment]
Hi nachifur,

We provide the script to generate the Adjusted ISTD. You can check the jupyter notebook in the data_processing folder.

Note that we keep the input images the same and only modify the shadow-free images.

**Comment by hieulem** [comment]
#11
